agent/src/main/java/com/teamscale/jacoco/agent/store/NoopUploader.java

```diff
@@ -0,0 +1,18 @@
+package com.teamscale.jacoco.agent.store;
+
+import java.io.File;
+
+/**
+ * Dummy uploader that does not provide any functionality. Can be used instead of null to mimic no upload behaviour.
+ */
+public class NoopUploader implements IUploader {
```

Comment:
noop is not really what it does though: it's an uploader that saves the xml files to disk. Thus, I'd prefer to call it `LocalDirectoryUploader` or something like that. It's an uploader that always succeeds and thus never deletes the cached files. This should be documented